### DISCUSSION TITLE
Nixos / home-manager and hyprland.conf

### DISCUSSION THREAD

**Discussion by ddaglenn** [hyprwm/Hyprland#1756] [General]
I've installed hyprland per the wiki using a flake.  I also have home manager managing my home.
Whenever I build my home.nix configuration I receive an error that my existing ~/.config/hypr/hyprland.conf is in the way of /nix/store/...home-manager-files/.config/hypr/hyprland.conf.

I am very new to Nixos.  Is there something I am overlooking in managing my hyprland.conf so I don't have to move my config every time I build?  Or is this more of a nixos question?

**Comment by notohh** [comment]
late answer, but you have to delete the pre-existing hyprland directory in .config, then `nixos-rebuild switch`, or `home-manager switch`, if you're using standalone home-manger.
